<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>957</width>
    <height>873</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Proyecto de  Computación 1</string>
  </property>
  <widget class="QLabel" name="milabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fichero de entrenamiento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="excel_cargar_button">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cargar excel</string>
   </property>
  </widget>
  <widget class="QTableView" name="mitableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>911</width>
     <height>191</height>
    </rect>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_Info">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Informacion de dataframe</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fichEntrenamiento_txt">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>fichero de carga</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="milabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Previsualización carga de datos para entrenamiento</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_Algor_RL">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>530</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Logistic regression</string>
   </property>
  </widget>
  <widget class="QLabel" name="milabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Selección de algoritmo de entrenamiento</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_Algor_KNN">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>610</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KNN - K nearest neighbor</string>
   </property>
  </widget>
  <widget class="QPushButton" name="entrenar_button">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>570</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Lanzar Entrenamiento</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rd_Algor_SVM">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SVM - support vectors machine</string>
   </property>
  </widget>
  <widget class="QTableView" name="mitableView_resultados">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>911</width>
     <height>191</height>
    </rect>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
